mosec/log.py

```diff
@@ -123,21 +141,23 @@ def use_log(level: int, formatter: logging.Formatter):
     return logger
 
 
-def use_pretty_log(level: int = logging.DEBUG):
+def use_pretty_log(level: int = logging.DEBUG, prefix: str = "", logger_name: str = ""):
     """Enable colorful log."""
-    return use_log(level, DebugFormat())
+    return use_log(level, DebugFormat(prefix=prefix), logger_name)
 
 
-def use_json_log(level: int = logging.INFO):
+def use_json_log(level: int = logging.INFO, prefix: str = "", logger_name: str = ""):
     """Enable JSON format log."""
-    return use_log(level, JSONFormat())
+    return use_log(level, JSONFormat(prefix=prefix), logger_name)
 
 
-def get_logger():
+def get_logger(internal: bool = False):
```

Comment:
This function is already exposed. We'd better not change the interface to avoid misuse. Can create a new one for internal usage.